src/acvp.c

```diff
@@ -455,6 +515,10 @@ ACVP_RESULT acvp_free_test_session (ACVP_CTX *ctx) {
                     acvp_cap_free_dsa_attrs(cap_entry);
                     free(cap_entry->cap.dsa_cap);
                     break;
+                case ACVP_KAS_ECC_TYPE:
+                    acvp_cap_free_kas_ecc_mode(cap_entry);
+                    free(cap_entry->cap.kas_ecc_cap);
```

Comment:
Possible to move this free into acvp_cap_free_kas_ecc_mode?

Reply:
sure.